elliott/elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -188,7 +188,10 @@ def verify_bugs_advisory_type(self, non_flaw_bugs, advisory_id_map, advisory_bug
         bugs_by_type = categorize_bugs_by_type(non_flaw_bugs, advisory_id_map, permitted_bug_ids=permitted_bug_ids)
```

Comment:
```suggestion
        bugs_by_type = categorize_bugs_by_type(non_flaw_bugs, advisory_id_map, permitted_bug_ids=permitted_bug_ids, permissive=True)
```
Can you add this as well? Recently we added this param to allow ignoring bugs that don't look right (like cve trackers). We want to have this so that the verification runs fully, otherwise errors short-circuit our verification flow.

Reply:
added in https://github.com/openshift-eng/art-tools/pull/292/commits/382f5459e2be12d8dd2ebdd9160efe47527af105